src/main/java/ua/lviv/iot/models/Color.java

```diff
@@ -0,0 +1,19 @@
+package ua.lviv.iot.models;
+
+public enum Color {
+

```

Comment:
Why would you use new lines after each enum constant?

Reply:
Fixed